source/php/Component/Segment/components/content.blade.php

```diff
@@ -1,78 +1,86 @@
-@if($title || $icon || $meta)
-@group([
-    'justifyContent' => 'space-between',
-    'alignItems' => 'start',
-    'gap' => 2,
-])
+@if ($title || $icon || $meta)
     @group([
-        'direction' => 'vertical'
+        'justifyContent' => 'space-between',
+        'alignItems' => 'start',
+        'gap' => 2
     ])
-    @if($title)
-        @typography([
-            "element" => "h2",
-            "variant" => ($layout == 'full-width') ? 'h1' : ($layout == 'card' ? 'h3' : 'h2'),
-            "classList" => [$baseClass . '__title'],
-            "autopromote" => true
+        @group([
+            'direction' => 'vertical'
         ])
-            {!! $title !!}
-        @endtypography
+            @if ($title)
+                @if ($link)
+                    @link([
+                        'href' => $link
+                    ])
+                    @endif
+                    @typography([
+                        'element' => 'h2',
+                        'variant' => $layout == 'full-width' ? 'h1' : ($layout == 'card' ? 'h3' : 'h2'),
+                        'classList' => [$baseClass . '__title', 'u-margin__bottom--1'],
+                        'autopromote' => true
+                    ])
+                        {!! $title !!}
```

Comment:
@link may be better placed here, to create a better markup.

Reply:
@sebastianthulin Do you mean inside @typography rather than the other way around?